src/integration/jackson/src/jvmMain/kotlin/community/flock/wirespec/integration/jackson/WirespecModule.kt

```diff
@@ -0,0 +1,64 @@
+package community.flock.wirespec.integration.jackson
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.core.JsonParser
+import com.fasterxml.jackson.core.JsonProcessingException
+import com.fasterxml.jackson.databind.BeanDescription
+import com.fasterxml.jackson.databind.DeserializationConfig
+import com.fasterxml.jackson.databind.DeserializationContext
+import com.fasterxml.jackson.databind.JsonDeserializer
+import com.fasterxml.jackson.databind.JsonNode
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.deser.BeanDeserializerModifier
+import com.fasterxml.jackson.databind.deser.std.StdDeserializer
+import com.fasterxml.jackson.databind.module.SimpleModule
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import community.flock.wirespec.Wirespec
+import java.io.IOException
+import kotlin.reflect.full.primaryConstructor
+
+class WirespecModule : SimpleModule() {
```

Comment:
This could use some documentation, including how to use it? Maybe the Jdk8Module can be of some inspiration

Reply:
I have used your inspiration